infrastructure/validators/build-a-community.md

```diff
@@ -235,8 +229,8 @@ appreciated in putting words and a person behind the name of someone running a s
 
```

Comment:
I would personally either reword this or remove this last part.

**Previous:** These kinds of gestures can be much
appreciated in putting words and a person behind the name of someone running a server.

**Suggested:** These kinds of gestures can be much appreciated and helps the nominator see a person behind the server.